tests/test_station.py

```diff
@@ -66,3 +66,17 @@ def test_get_table():
         if value is not '-':
             assert value in mm2.measurements.properties[4].values()
 
+
+def test_get_names():
```

Comment:
after updating `names `function as for Stephen's request then you can probably change this test to be for `names` and maybe leave few tests for the private `_get_names` function just to be safe

can you add tests also for when `measurement_type_id=None`?

Reply:
we decided to keep `names` function as it is and change `get_names` to be public, so ok to have tests for get_names